june-2019/mybb-quarterly-review-q2-2019.md

```diff
@@ -0,0 +1,38 @@
+# MyBB Quarterly Review - Q2 2019
+
+Hello, and welcome to the first MyBB Quarterly Review blog post.
+
+You may be wondering what this means? We realise that we've been pretty poor at keeping the community abreast of what's going on in the project recently, which is something we want to rectify. Our plan from here on out is a continuous series of blog posts — one every quarter — that cover the goings on within an around MyBB. These posts won't only be focused upon the MyBB core, but will also highlight notable plugins, themes and user contributions.
+
+As this is new to all of us, I'm sure it'll take some time to get used to and to settle into a consistent format. We'd welcome any feedback on the kind of content you'd like to see in these posts and ways we can improve the way we present the project and what's going on!
```

Comment:
Where should this feedback be sent? Comments on the blog post, community forums (which forum?)...